CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@ Add: httpsPrepare.sh script to contextBroker-tests RPM (Issue #2)
 Fix:  Fixed a bug about generic error handling for API version 2 (Issue #1087)
 Fix:  Fixed a bug about error handling of invalid Service-Path for API version 2 (Issue #1092)
 Add:  Implemented a RAM-cache for patterned ngsi10 subscriptions (Issue #1048)
+Add:  POST /v2/entities/<entityId> entity creation under version 2 (Issue #981)
```

Comment:
Actually, it is not entity creating but append/update attributes in an existing entity, isn't it?